index.ts

```diff
@@ -228,14 +231,20 @@ const cutTranscriptionEventHandler = async (event: TranscriptionEvent) => {
 
 const responseReflexEventHandler = async (): Promise<void> => {
   await globalWhisperPromise;
-  const responseReflexEvent: ResponseReflexEvent = {
-    timestamp: Number(Date.now()),
-    eventType: 'responseReflex',
-    data: {
-      transcription: getTransciptionSoFar()
+
+  var transcription = getTransciptionSoFar();
```

Comment:
avoid var; use const

Reply:
https://github.com/yacineMTB/talk/issues/42

Reply:
aye, is `let` fine as well? reading about it and I guess avoiding var is to avoid hoisiting with default values?